recipes/aihwkit/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+aihwkit_variant:
+  - cpu
+  - gpu
```

Comment:
You don't need this. You can use `cuda_compiler_version == "None"` for the CPU version.

Reply:
Thank you for the review comment.  I removed the conda_build_config.yaml and updated the meta.yaml and build.sh accordingly.   Thanks again.

Reply:
I pushed out the changes.  The PR was built successfully.  Would you please review the PR?  Thanks a million.